src/librustdoc/html/static/main.js

```diff
@@ -113,7 +113,8 @@
             from = parseInt(match[1], 10);
             to = Math.min(50000, parseInt(match[2] || match[1], 10));
             from = Math.min(from, to);
-            if (document.getElementById(from).length === 0) {
+            var elem = document.getElementById(from);
+            if (!elem || elem.length === 0) {
```

Comment:
Check for `elem.length === 0` is redundant, since `getElementById` always returns a single element since there can be only one element with a given ID. There's no way you could get an array or NodeList of elements from this method. See https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById